docs/src/assets/scss/syntax-highlighter.scss

```diff
@@ -118,6 +118,7 @@ pre[class*="language-"] {
     font-variant-numeric: tabular-nums;
     text-align: right;
     width: 1.2em;
+    line-height: 0;
```

Comment:
Mind sharing which browser and its version as well as the OS to reproduce the problem?

Reply:
I am seeing this problem in
 **Browser** - Firefox
**Version** - 108.0.1 (latest)
**Operating system** - Windows 10 Pro (64-bit)

I am not getting this problem in **Chrome** browser but this fix (PR) is not affecting that

Reply:
Thanks, I can reproduce it in Firefox.

Seems the `line-height` for `code` has no effect under firefox, I'll see why it happens.

Actually I can find the problem even before my pull request (screenshots from https://eslint.org/docs/latest/rules/array-callback-return under firefox):

<img width="669" alt="" src="https://user-images.githubusercontent.com/1091472/208118609-c4ea1888-5e1a-4bf3-9ff3-4326e47c9c11.png">

<img width="557" alt="" src="https://user-images.githubusercontent.com/1091472/208118758-51b0f3e1-4897-4b5f-bb30-2efe631a204b.png">

My pull request happened to expose it :)

Reply:
So I compared the css rules before to those after my pull request and found there was one rule missing from my pr. Applying that rule to `.line-numbers-wrapper` seems to fix the problem under firefox:

```
.line-numbers-wrapper {
  font-variant-ligatures: none;
}
```
Could you try it and verify if it works for you? I reckon it doesn't make sense to set `line-height` to `0`, though it did work in this case under firefox.

Reply:
Yes, it is working for me,
ok then, i am closing this PR issue is all yours

Reply:
Thanks, I've updated that pull request to fix this issue.